### PR TITLE
Add FFI support for enclave quoting

### PR DESCRIPTION
**Pull request by nick-mobilecoin**
Adds the FFI for the quoting and attestation for DCAP.

### CODE REVIEW

**Comment by nick-mobilecoin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/2" target="_blank">Graphite</a> 
    * **PR #11** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/11" target="_blank">Graphite</a> 
      * **PR #14** <a href="https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/14" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/mobilecoinfoundation/sgx/14?utm_source=stack-comment).

File: untrusted/dcap/sys/types/build.rs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
+    bindings
+        .write_to_file(out_path.join("bindings.rs"))
+        .expect("Couldn't write bindings!");
+}
+
+fn include_verify_if_configured(builder: Builder) -> Builder {
+    #[allow(unused_mut)]
+    let mut builder = builder;
```
Comment (remoun) [inline untrusted/dcap/sys/types/build.rs:31]: IIUC we can just make the `builder` param `mut`
```suggestion
#[allow(unused_mut)]
fn include_verify_if_configured(mut builder: Builder) -> Builder {
```

Comment (nick-mobilecoin) [reply untrusted/dcap/sys/types/build.rs]: converted as suggested

**Comment by nick-mobilecoin** [comment]
[generate_bindings.txt](https://github.com/mobilecoinfoundation/sgx/files/8882406/generate_bindings.txt)
[verify_bindings.txt](https://github.com/mobilecoinfoundation/sgx/files/8882407/verify_bindings.txt)
[bindings.txt](https://github.com/mobilecoinfoundation/sgx/files/8882409/bindings.txt)
THe generated bindings files as txt :(

**Comment by nick-mobilecoin** [comment]
Closing as this came in via #53

**Pull request closed without merging**